File: Pony_Racing/core/src/com/nopalsoft/ponyrace/objetos/DynamicGameObject.java
package com.nopalsoft.ponyrace.objetos;

import com.badlogic.gdx.math.Vector3;

public class DynamicGameObject extends GameObject {
    public final Vector3 velocity;

    public DynamicGameObject(float x, float y, float z, float[] vertices) {
        super(x, y, z, vertices);
        velocity = new Vector3();
    }
}

File: Galaxy_Invaders/core/src/com/nopalsoft/invaders/game_objects/DynamicGameObject.kt
package com.nopalsoft.invaders.game_objects

import com.badlogic.gdx.math.Vector2

open class DynamicGameObject : GameObject {
    @JvmField
    val velocity: Vector2

    constructor(x: Float, y: Float, width: Float, height: Float) : super(x, y, width, height) {
        velocity = Vector2()
    }

    constructor(x: Float, y: Float, radio: Float) : super(x, y, radio) {
        velocity = Vector2()
    }
}
